lib/provider/bpmn/parts/implementation/ErrorEventDefinition.js

```diff
@@ -19,7 +19,7 @@ module.exports = function(group, element, bpmnFactory, errorEventDefinition, tra
   group.entries = group.entries.concat(
     elementReferenceProperty(element, errorEventDefinition, bpmnFactory, translate, {
       id: 'error-element-name',
-      label: translate('Global Error Name'),
+      label: translate('Name'),
```

Comment:
the labels differ from the sketch in the referenced issue (https://github.com/bpmn-io/bpmn-js-properties-panel/issues/422)

![image](https://user-images.githubusercontent.com/9433996/109947874-157a8d80-7cda-11eb-9941-ae854a736047.png)

but I guess the sketch got outdated in the meanwhile.

Reply:
The sketch is outdated yeah, we settled for naming without the "Error-" prefix.